src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
because key and value the same, you could simply write
```suggestion
    name,
    wheels,
    version,
```